portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,6 +37,16 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
I recommend to move string constants "timestamp" and "content" to class level and share them between both methods.

Reply:
Done.

Reply:
Sorry, I mean the name of the properties, not the variables to store the content.

In general, you should avoid hardcoded string that appears multiple times in your code. This can help you:
1. avoid typo. e.g.  setProperty("content",A), then getProperty("contant") returns empty result
2. make change safer and easier, e.g. you later decide to change the property name from "content" to "text", you don't have to go through the whole code to find the hardcoded string.

Reply:
To make sure, do you mean something like this:
final String comment = "comment";
final String timestamp = "timestamp";

Reply:
Right, internally we usually write it like:
private static final String CONTENT_TEXT_PROPERTY_NAME = "content";

Reply:
Ok, I will implement that, thank you!